Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN apt-get update && apt-get install -y gnupg2 --no-install-recommends curl &&
     curl -sL http://repo.radeon.com/rocm/rocm.gpg.key | apt-key add -
 
 # Add rocm repository
-RUN sh -c 'echo deb [arch=amd64 trusted=yes] http://repo.radeon.com/rocm/apt/5.7/ focal main > /etc/apt/sources.list.d/rocm.list'
+RUN sh -c 'echo deb [arch=amd64 trusted=yes] http://repo.radeon.com/rocm/apt/6.0/ focal main > /etc/apt/sources.list.d/rocm.list'
```

Comment:
The version of rocm needs to be updated in all docker files.

Reply:
Also, I think a unit test wont pass with 6.0 due to a bug in hiprtc. I think this needs to use 6.0.1.